app/models/github/team.rb

```diff
@@ -0,0 +1,27 @@
+class Github::Team
```

Comment:
@johndbritton I'm trying my hand at the `NullObject` pattern for the GitHubTeams. What do you think of this?